src/css/profile/wearable/changeable/theme-circle/popup.less

```diff
@@ -368,44 +362,67 @@
 		/****************************************
 		Popup Toast Small
 		****************************************/
+		//toast small animation
+		@keyframes toastsmall {
+			.transition-toastsmall();
+		}
+		.transition-toastsmall() {
+			from {
+				opacity: 0;
+			}
+			100% {
+				opacity: 1;
+			}
+		}
 
 		&.ui-popup-toast-small {
 			height: 160 * @unit_base;
```

Comment:
Wouldn't it be great to change all unit_base to px_base?

Reply:
In case of wearable profile, normally use  both unit_base and px_base.
We'll organize the units at a time later.